i18n/fi.po

```diff
@@ -0,0 +1,1222 @@
+"    mailinglist, or you can even send us raw patches --- but it will be more "
+"work\n"
+"    for us to integrate it."
+msgstr ""
+"Githubin talletuspyyntö on meille mieluisin tapa, koska se tekee meille "
+"helpoksi\n"
+"    arvioida ja yhdistää lähdekoodia. Joten,\n"
+"    voit lisäksi kloonata viimeisimmän git-säilön (ks. linkki oikealla), "
+"valmistella haaran jossa\n"
+"    koodisi sijaitsee, ladata sen jonnekin (esimerkiksi <a\n"
+"                                                   href=\"http://github.com/"
+"\">github</a>) ja lähettää meille linkin <a\n"
+"                                                                                                                      href="
+"\"https://groups.google.com/forum/#!forum/sympy\">sympy</a>\n"
+"    postituslistalle, tai voit jopa lähettää meille raakamuutostiedostoja "
+"--- mutta se tietää meille työtä\n"
+"    koska meidän pitää integroida muutokset."
+
+#: templates/development.html:65
+msgid ""
+"SymPy has a broad <a href=\"http://docs.sympy.org/dev/aboutus.html#sympy-"
+"development-team\">development team</a>, send us a patch and your name will "
+"appear there too."
+msgstr ""
+"SymPy-projektilla on laaja <a href=\"http://docs.sympy.org/dev/aboutus."
+"html#sympy-development-team\">kehitystiimi</a>, lähetä meille "
+"korjaustiedosto ja sinun nimesi ilmestyy kehitystiimiimme."
+
+#: templates/development.html:69
+msgid "Sources of the Webpages"
+msgstr "Sivustojen lähdekoodi"
+
+#: templates/development.html:72
+msgid "Sources of these webpages are at"
+msgstr "Näiden sivujen lähdekoodi on osoitteessa"
+
+#: templates/development.html:75
+msgid "Feel free to send us patches with improvements."
+msgstr "Voit lähettää meille parannuksia sisältäviä muutostiedostoja vapaasti."
+
+#: templates/development.html:84
+msgid "Git Repository"
+msgstr "Git-repo"
+
+#: templates/development.html:86 templates/download.html:21
+msgid "Main git repository"
+msgstr "Gitin pääsäilö"
+
+#: templates/development.html:90
+msgid "Online access"
+msgstr "Online-pääsy"
+
+#: templates/docs.html:11
+#, python-format
+msgid ""
+"The documentation is maintained using the <a href=\"%(url)s\">Sphinx</a> "
+"Python documentation generator."
+msgstr ""
+"Dokumentaatiotamme ylläpidetään käyttäen <a href=\"%(url)s\">Sphinx</a> "
+"Python dokumentaatiogeneraattoria."
+
+#: templates/docs.html:13
+msgid "You can find it at"
+msgstr "Voit löytää sen osoitteesta"
+
+#: templates/donate.html:8
+msgid "Support SymPy Development"
+msgstr "Tue SymPyn kehitystä"
+
+# xgettext:no-python-format
+#: templates/donate.html:11
+#, python-format
+msgid ""
+"SymPy will always be 100% open source software, free for all to use\n"
+"and released under the liberal terms of the modified BSD license.\n"
+msgstr ""
+"SymPy tulee aina olemaan 100% avoimen lähdekoodin ohjelmisto, vapaa kaikkien "
+"käytettäväksi\n"
+"ja julkaistu muokatun BSD-lisenssin sallivilla ehdoilla.\n"
```

Comment:
That is something I suggested instead of "liberaaleilla termeillä". Perhaps the words should be reordered: "sallivilla muokatun BSD-lisenssin ehdoilla".